bot/meta/logger.py

```diff
@@ -48,14 +48,14 @@ def filter(self, record):
 @cmd_log_handler
 def log(message, context="GLOBAL", level=logging.INFO, post=True):
```

Comment:
Probably best to just test for None and return prematurely, or even raise a ValueError. If we are passing `None` to the logger message something is very wrong.